tests/unit/test_noise_form.py

```diff
@@ -113,7 +113,7 @@ def test_noise_order(mocker, dummy_data, dummy_config_noise_numbers):
     for field in NOISE_TYPES._fields:
         mock_return = (
             dummy_data[["event_type"]]
-            if field in ["omission", "duplication   "]
+            if field in ["omission", "duplication"]
```

Comment:
is `"omission"` still what we want here?

Reply:
+1

Reply:
It is (for now). There's going to be a discrepancy for a bit between the field name "omission" and the front-facing key name "omit_rows". I did create a ticket to clean that up